internal/updater/providers/slickvpn/servers.go

```diff
@@ -0,0 +1,72 @@
+/*
+Package slickvpn contains code to obtain the server information for the SlickVPN provider.
+Note: As far as I'm aware, SlickVPN only provide UDP config files.
+If we want to implement TCP, we would need to perhaps take the UDP config file and adjust
+its config to match SlickVPN's TCP settings (which I believe that they do support)
+*/
+package slickvpn
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/qdm12/gluetun/internal/models"
+	"github.com/qdm12/gluetun/internal/updater/resolver"
+)
+
+var ErrNotEnoughServers = errors.New("not enough servers found")
+
+func GetServers(ctx context.Context, client *http.Client,
+	presolver resolver.Parallel, minServers int) (
+	servers []models.Server, warnings []string, err error) {
+	udpHostToURL, err := getHostToURL(ctx, client)
```

Comment:
Would it be possible to combine both `getHostToURL` and `fetchLocationInfo` together? They look like they could combine the same network call, and their code looks rather similar.